Model/Model.tex

```diff
@@ -893,7 +893,7 @@ \subsection{Global variables}
 The following code displays some information in a new of the \ct{Transcript}.
 
 \begin{code}{}
-Transcript show: 'Squeak is fun and powerful'; cr

```

Comment:
:D I have already also done that change once but rhi objected. But as we are now two votes: keep it.

Reply:
I'm interested, what was the argument against `showln:`? That it prints the cr first?

By "keep it", do you mean "keep it as on master" or "keep it as in this PR"? 😅

Reply:
@LinqLover Keep it as in this PR. (keep the change :D)